package.json

```diff
@@ -20,26 +20,15 @@
     "version": "node ./scripts/version.mjs",
     "publish": "pnpm build && pnpm changeset publish"
   },
-  "dependencies": {
-    "@deck.gl/core": "~8.8.27",
-    "@deck.gl/geo-layers": "~8.8.27",
-    "@deck.gl/layers": "~8.8.27",
-    "@deck.gl/react": "~8.8.27",
-    "@luma.gl/constants": "~8.5.21",
-    "@luma.gl/core": "~8.5.21",
-    "@luma.gl/shadertools": "~8.5.21",
-    "@luma.gl/webgl": "~8.5.21"
-  },
```

Comment:
Why did we ever have deps in the root package?

Reply:
Not sure. I think vestigial from before the package meta script.